### PR TITLE
Add downloaded dataset metric

### PR DESCRIPTION
**Pull request by dmgaldi**
Added a metric for dataset download usage by study ID.

Resolves https://github.com/VEuPathDB/service-dataset-download/issues/5

Tested locally, verified metrics get emitted.

### CODE REVIEW

File: src/main/java/org/veupathdb/service/dsdl/Metrics.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package org.veupathdb.service.dsdl;
+
+import io.prometheus.client.Counter;
+
+public class Metrics {
+  private static final Counter DOWNLOADS_BY_DATASET = Counter.build()
```
Comment (ryanrdoherty) [inline src/main/java/org/veupathdb/service/dsdl/Metrics.java:6]: You've got to be careful using the words "dataset" vs "study".  The label you're attaching to the increment is a study ID e.g. PRISM0001-1 but I think if people are browsing prometheus/grafana and see dataset name, they will expect to see counts broken down by dataset e.g. DS_0ad509829e.  Maybe pick one?  Study is probably better, but you might want to ask someone.... don't know who needs this?

**Review by ryanrdoherty — CHANGES_REQUESTED** [review]

File: src/main/java/org/veupathdb/service/dsdl/Metrics.java (hunk 1 above)
Comment (dmgaldi) [reply src/main/java/org/veupathdb/service/dsdl/Metrics.java]: I updated to use study to be consistent with the rest of the API's terminology

**Review by ryanrdoherty — APPROVED** [review]
Looks good.

**Comment by dmgaldi** [comment]
I think this was merged manually and it somehow didn't get picked up by github

**Pull request closed without merging**